src/style.css

```
@@ -1 +1,57 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
```

point 5 https://mateacademy.slack.com/archives/C02LA8ED589/p1637236987052200

```suggestion
* {
  margin: 0;
}
```